packages/main/src/Input.js

```diff
@@ -581,17 +585,38 @@ class Input extends UI5Element {
 	}
 
 	get ariaDescribedBy() {
+		if (this._accInfo && this._accInfo.ariaDescribedBy) {
+			return `${this.suggestionsTextId} ${this.valueStateTextId} ${this._accInfo.ariaDescribedBy}`.trim();
+		}
 		return `${this.suggestionsTextId} ${this.valueStateTextId}`.trim();
 	}
 
 	get ariaHasPopup() {
+		if (this._accInfo && this._accInfo.ariaHasPopup) {
```

Comment:
You can do one check less, as undefined is fine:

if (this._accInfo) {
			return this._accInfo.ariaHasPopup;
	}